metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -540,19 +563,24 @@
                            tmp.concept_id = t1.concept_id AND
                            tmp.revision_id = t1.revision_id)"
                           table)]
-            efs-delete (when (not (= "efs-off" (efs-config/efs-toggle)))
+            efs-delete (when (not= "dynamo-off" (dynamo-config/dynamo-toggle))
```

Comment:
`efs-delete` and `dynamo-delete` are true on the same condition, so they could be merged